source/includes/gcp/_k8_statefulsets.md

```diff
@@ -0,0 +1,334 @@
+                  }
+                ],
+                "readinessProbe": {
+                  "failureThreshold": 3,
+                  "initialDelaySeconds": 15,
+                  "periodSeconds": 10,
+                  "successThreshold": 1,
+                  "tcpSocket": {
+                    "port": 3000
+                  },
+                  "timeoutSeconds": 1
+                },
+                "resources": {},
+                "terminationMessagePath": "/dev/termination-log",
+                "terminationMessagePolicy": "File",
+                "volumeMounts": [
+                  {
+                    "mountPath": "/etc/aerospike",
+                    "name": "config-volume"
+                  }
+                ]
+              }
+            ],
+            "dnsPolicy": "ClusterFirst",
+            "restartPolicy": "Always",
+            "schedulerName": "default-scheduler",
+            "securityContext": {},
+            "terminationGracePeriodSeconds": 30,
+            "volumes": [
+              {
+                "configMap": {
+                  "defaultMode": 420,
+                  "items": [
+                    {
+                      "key": "aerospike.conf",
+                      "path": "aerospike.conf"
+                    }
+                  ],
+                  "name": "test-aerospike"
+                },
+                "name": "config-volume"
+              }
+            ]
+          }
+        },
+        "updateStrategy": {
+          "rollingUpdate": {
+            "partition": 0
+          },
+          "type": "RollingUpdate"
+        }
+      },
+      "status": {
+        "collisionCount": 0,
+        "currentReplicas": 1,
+        "currentRevision": "test-aerospike-6db7776c7d",
+        "observedGeneration": 1,
+        "readyReplicas": 1,
+        "replicas": 1,
+        "updateRevision": "test-aerospike-6db7776c7d",
+        "updatedReplicas": 1
+      }
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/statefulsets</code>
+
+Retrieve a list of all stateful sets in a given [environment](#administration-environments).
+
+| Attributes                                 | &nbsp;                                                                |
+| ------------------------------------------ | --------------------------------------------------------------------- |
+| `id` <br/>*string*                         | The id of the stateful set                                            |
+| `metadata` <br/>*object*                   | The metadata of the the stateful set                                  |
+| `metadata.creationTimestamp` <br/>*string* | The date of creation of the stateful set as a string                  |
+| `metadata.labels` <br/>*map*               | The labels associated to the stateful set and there associated values |
```

Comment:
The wording here is a little awkward to me. The  way that the word "associated" is repeated does not sit with me, a particularly pedantic English minor. So, you know, grain of salt...

Also, use remember to use the possessive "their", not "there" here and on subsequent lines ;)

Reply:
nice catch!

Reply:
I removed the "and..." part. I copy-pasted it from the official docs without proofreading 🤦  8a67d85acf842bb3127d200f31526fd28cf7a834

Reply:
Even worse... it wasn't the official docs. I copy-pasted from another file :p 

I fixed the all here 2ae8873e6165bc3851499878a98e8ead9c7decc5